### PR TITLE
Upgrade to Storybook v4

### PR DESCRIPTION
**Pull request by mlaco**

### CODE REVIEW

File: android/app/src/main/java/com/chainreactapp/MainApplication.java [hunk 1]
```diff
@@ -32,6 +32,7 @@ public boolean getUseDeveloperSupport() {
     protected List<ReactPackage> getPackages() {
       return Arrays.<ReactPackage>asList(
         new MainReactPackage(),
+        new AppCenterReactNativePackage(MainApplication.this),
```
Comment (ryanlntn) [inline android/app/src/main/java/com/chainreactapp/MainApplication.java]: You probably don't want this duplicate.

File: android/app/build.gradle [hunk 2]
```diff
@@ -150,6 +150,7 @@ android {
 }
 
 dependencies {
+    compile project(':appcenter')
```
Comment (robinheinze) [inline android/app/build.gradle]: Do you know why there are changes here? I wouldn't expect build.gradle to change with a Storybook upgrade. Also, I think this line would be redundant to line 156 below, which uses the new `implementation` syntax.

Comment (ryanlntn) [inline android/app/build.gradle]: You probably don't want this duplicate either.

File: android/app/src/main/java/com/chainreactapp/MainApplication.java (hunk 1 above)
Comment (robinheinze) [inline android/app/src/main/java/com/chainreactapp/MainApplication.java]: Same here.

**Comment by robinheinze** [comment]
Hah, @ryanlntn beat me to it!

**Comment by robinheinze** [comment]
Were you able to get the storybooks to run in the app after the upgrade? If so, can you include a screenshot?

File: android/app/build.gradle (hunk 2 above)
Comment (mlaco) [reply android/app/build.gradle]: I removed this, but it appeared when I ran `react-native link appcenter`

File: .circleci/config.yml [hunk 3]
```diff
@@ -7,7 +7,7 @@ jobs:
   build:
     docker:
       # specify the version you desire here
-      - image: circleci/node:7.10
+      - image: circleci/node:8.9
```
Comment (robinheinze) [inline .circleci/config.yml]: There's a couple new commits on master that include this change as well. You might try merging/rebasing.

Comment (mlaco) [reply .circleci/config.yml]: Actually, I did this. An error like this occurred on the CircleCI build:

`error css-loader@1.0.1: The engine "node" is incompatible with this module. Expected version ">= 6.9.0 <7.0.0 || >= 8.9.0".`

I was trying to fix this by configuring Circle build to use Node 8.9.0.

**Pull request merged**